Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -1930,17 +1930,17 @@ CustomPasteboardDataEnabled:
 
 CustomStateSetEnabled:
   type: bool
-  status: testable
+  status: stable
   category: dom
   humanReadableName: "CustomStateSet"
```

While we're there can we rename this?

```suggestion
  humanReadableName: "CustomStateSet and :state() pseudo-class"
```